docs/glossary.md

```diff
@@ -0,0 +1,25 @@
+# Glossary
+
+
+## Agent
+## Bundle
+## Chart
+## Cluster
+## Continuous Delivery
+## Custom Resource
+## Custom Resource Definition
+## Deployment
+## Downstream Cluster
+## fleet.yaml
+## GitOps
+## GitRepo
+## Label
+## Multi-Cluster
+## Namespace
```

Comment:
```suggestion
## Namespace
## Reconcile

Reconciling is used in the context of states in Kubernetes clusters. Reconciling a resource means updating it so that its _actual_ state matches its _expected_ state, be it from configuration, eg. from a git repository, chart, etc.

When using GitOps, updates to a git repository may translate into new expected states for resources configured through that git repository. As a result, affected resources will be reconciled.
A resource's state may also depend on another resource, leading to additional reconciliation. For instance, a cluster group's status depends on statuses of individual clusters contained in that cluster group. Therefore, a change in a cluster's state will result in any cluster group(s) to which that cluster belongs to being reconciled as well.
```